Makefile

```diff
@@ -2,6 +2,7 @@ T=openssl
 
 PREFIX		?=/usr/local
 LIB_OPTION	?= -shared 
+CC			 = gcc
```

Comment:
In fact, I had it "properly" indented, with spaces only, but that meant changing the previous two lines as well. The problem here is that, as soon as the width of tab changes (mine is set to 4), the alignment changes. That's why I don't like to align with tabs between parts of text (I only use tabs for left-side indentation).
So, how do you want me to indent?